pkg/services/ngalert/notifier/status.go

```diff
@@ -1,16 +1,19 @@
 package notifier
 
 import (
+	"encoding/json"
+
 	apimodels "github.com/grafana/grafana/pkg/services/ngalert/api/tooling/definitions"
 )
 
+// TODO: We no longer do apimodels at this layer, move it to the API.
```

Comment:
No issue as is part of a very large, very obvious effort - don't have a concrete plan yet will create one once I do.